src/register.c

```diff
@@ -2073,7 +2128,17 @@ do_put(
 	else
 	{
 	    linenr_T	new_lnum = new_cursor.lnum;
-	    size_t	len;
+	    int		indent;
+	    int		orig_indent;
```

Comment:
```suggestion
	    int		orig_indent = 0;
```

and then get rid of the lines below:
```
    else
		orig_indent = 0;
```